backend/lib/models/Post.js

```diff
@@ -9,13 +9,13 @@ const POST_TYPES = [
   "Education",
   "Entertainment",
   "Funding",
-  "Groceries/Food",
+  "Groceries / Food",
```

Comment:
Why adding these spaces with slashes at categories?
Just to know because if we follow this need to update the data model docs also

Reply:
I changed this so it would match the frontend post-type value to allow PATCH/edits to a post. If it is going to be too much a hassle to update the docs, I can just change this property value in the frontend or do a mapping, no worries.

Reply:
You'll need to merge/rebase on master to resolve conflicts but anyways and you will see we recently updated filterOptions to match backend model - https://github.com/FightPandemics/FightPandemics/blob/master/client/src/assets/data/filterOptions.js. 

Ideally we'd have a shared/common constants like below. Might be a good post-MVP project to initiate? Keep model and as-is and update front-end options to match model.

```javascript
const POST_TYPES = {
INFORMATION: "Information",
LEGAL: "Legal",
}
```
```

Reply:
It is easy to update it at documentation, just have to watch out to keep it consistent across the app.
That seems nice @joshmorel

Reply:
Updating docs is easy, just have to watch out to keep it consistent across the app.
Also, that sounds good @joshmorel

Reply:
I'll do the mapping in the frontend. Don't change the doc @Naraujo13 The mapping should be super easy since I know which fields to target. Thank you both @Naraujo13 @joshmorel ❤️